src/store.js

```diff
@@ -127,6 +127,66 @@ const store = new Vuex.Store({
 
     testAPI() {
       return $http.get()
+    },
+
+    getSeatingLayouts() {
+      // return $http.get('seatingPlan')
+      // .catch(err => {
+      //   err.message = "Could not get seating layouts. Please try again later"
+      //   throw err
+      // })
+
+      // Return mock data since api is not here yet
+      return new Promise((resolve) => {
+        resolve({
+          seatingLayouts: [
```

Comment:
Good idea to include mock data, in the event that things do not get completed for demo tomorrow